data.sql

```diff
@@ -15,3 +15,28 @@ VALUES('Charmander', '2020-02-08', 0, true, -11),
     ('Ditto', '2022-05-14', 4, true, 22);
 //insert skipped animal data
 INSERT INTO ANIMALS(NAME,DATE_OF_BIRTH,ESCAPE_ATTEMPTS,NEUTERED,WEIGHT_KG) VALUES('Squirtle','1993-04-02',3,false,-12.13);
+//insert data into owners table;
+INSERT INTO OWNERS(FULL_NAME,AGE)
+ VALUES('Sam Smith',34),
+ ('Jennifer Orwell',19),
+ ('Bob',45),
+ ('Melody Pond',77),
+ ('Dean Winchester',14),
+ ('Jodie Whittaker',38);
+ //insert data into species table;
+ INSERT INTO SPECIES (NAME) VALUES('Pokemon'),('Digimon');
+
+//Modify your inserted animals so it includes the species_id value:
+//If the name ends in "mon" it will be Digimon
+//All other animals are Pokemon
+UPDATE ANIMALS SET species_id=2 WHERE TRIM(NAME) LIKE '%mon';
+//Sam Smith owns Agumon.
```

Comment:
- [ ] Well done @ismayito, you did a great job by adding the query to update the animals but it seems like you removed the query for updating the `species_id = 1` animals ends with %mon in name, now with an above-selected query, in the database all animals species_id will set to `2`. But according to the project requirement that I also have added after this comment as a reference, you should set the animal species_id with 1 (Digimon), whose name ends with `%mon` and the rest of the animals should be set to (pokemon) `species_id` with 2. Don't worry I have added the bullet points with pseudo below. In short, both your queries should be in the `data.sql` file and the PR description with screenshots.

### Requirement

![image](https://github.com/ismayito/vet-clincic-database/assets/66476329/c0885738-00b2-4815-8971-b4f2f2ea25d9)

### Pseudo

1. Set the animals species_id = 1 where the name of the animals ends with "%mon" which means the last three characters are mon. (It will update the majority of the animals)
2. Set the rest of the animals with species_id = 2, where animals species_id is null. (Those animals will get the species_id = 2 which does contain the default value `Null` in the table)